src/doc/unstable-book/src/conservative-impl-trait.md

```diff
@@ -6,5 +6,19 @@ The tracking issue for this feature is: [#34511]
 
 ------------------------
 
+The `conservative_impl_trait` feature allows a conservative form of abstract return types.
```

Comment:
Probably because I'm new to the language, but this does not make it clearer to me. The example also does not make me understand why regular stable Rust would not behave this way